Docs: Add hide_edition config to the docs (#73576)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

# Set to complete URL to override loading logo
;loading_logo =

> # Hides the Grafana edition being used in the footer if set to "true"

Reviewer: ```suggestion
# Set to `true` to remove the Grafana edition from appearing in the footer.
```

Author: I've used the text here but removed the full stop because none of the other configs have full stop in their comment either (this block comes directly from code)
